### PR TITLE
Suggest Draft Sec+Iss_Temp

### PR DESCRIPTION
**Pull request by kk5dire**
Draft Security Policy and issue templates.

### CODE REVIEW

**Comment by kk5dire** [comment]
> Draft Security Policy and issue templates.

Security policy link-back to issues page.
2 Issue templates added

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+
+| Version | Supported          |
+| ------- | ------------------ |
+| [0.0.2.4](https://ticketsbot.net/)  | :white_check_mark: |
```
Comment (Tech-TTGames) [inline SECURITY.md:8]: Why link to ticketsbot?

Comment (kk5dire) [reply SECURITY.md:8]: > Why link to ticketsbot?

Nothing else in mind to put atm, Just a *something*

Comment (kk5dire) [reply SECURITY.md:8]: If preferred, you would be able to change/remove it, as there is no "official" supported/unsupported version/s of the bot.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Supported Versions
+
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.0.2.4+  | :white_check_mark: |
```
Comment (kk5dire) [inline SECURITY.md:8]: Removed link-back to ticketsbot.net

**Comment by kk5dire** [comment]
Assuming no other comments due to lack of additional reviews.

**Pull request merged**